pages/documentation/index.html

```diff
@@ -184,17 +184,17 @@ <h5>XIIDM</h5>
         </div>
     </div>
 
-    <!-- PSS/E -->
+    <!-- PSS®E -->
     <div class="col-lg-4 col-sm-6 my-2">
         <div class="media documentation_item">
             <div class="doc-icon align-self-start">
                 <img class="doc-icon" src="/assets/img/crystal/crystal_clear_filesystem_folder_html.png" alt="">
```

Comment:
I think we can change the icon too?

Reply:
I am not sure we could use an image or icon directly related to the Siemens product

Reply:
I've added an image taken from the public PSS®E brochure, respecting colours and font